src/Network/Network/NetworkManager/NetworkGroup/NewAzNetworkGroupCommand.cs

```diff
@@ -60,6 +60,16 @@ public class NewAzNetworkGroupCommand : NetworkGroupBaseCmdlet
          HelpMessage = "Description.")]
         public virtual string Description { get; set; }
 
+        [Parameter(
+           Mandatory = false,
+           ValueFromPipelineByPropertyName = true,
+           HelpMessage = "Network Group member type. Valid values include 'VirtualNetwork' and 'Subnet'.")]
+        [ValidateSet(
+            MNM.GroupMemberType.VirtualNetwork,
+            MNM.GroupMemberType.Subnet,
+            IgnoreCase = true)]
+        public string MemberType { get; set; }
```

Comment:
`MemberType` is a new input parameter for the cmdlet and help doc is not updated